docker/Dockerfile

```diff
@@ -12,37 +13,39 @@ RUN apt update && \
 # move policy file
 RUN mv /etc/ImageMagick-6/policy.xml /etc/ImageMagick-6/policy.xml.off
 
-# install pip and Python pkgs
+# install pip and Python pkg
 # RUN pip3 install -U pip && \
 #     pip install amc2moodle
 WORKDIR /tmp
 RUN git clone https://github.com/nennigb/amc2moodle.git -b master
 WORKDIR /tmp/amc2moodle
-RUN pip3 install .
+RUN pip3 install -U pip && \
+    pip3 install .
 
 # check if amc2moodle and moodle2amc work
 WORKDIR /
 RUN python -m amc2moodle.amc2moodle.test && \
     python -m amc2moodle.utils.test
```

Comment:
The moodle2amc tests are not present here. Please find the CI lines
```
    - name: Test moodle2amc
      run: |
        # add automultiplechoice.sty local copy to LaTeX PATH (for this step)
        export TEXINPUTS=.:./amc2moodle/moodle2amc/test:$TEXINPUTS
        python -m amc2moodle.moodle2amc.test
```

Reply:
I have added it.